vm/src/stdlib/math.rs

```diff
@@ -18,10 +18,24 @@ use rustpython_common::float_ops;
 use std::cmp::Ordering;
 
 // Helper macro:
+macro_rules! result_or_overflow {
+    ( $value:ident, $result:ident, $vm: ident ) => {
+        if !$result.is_finite() && $value.is_finite() {
+            // CPython doesn't return `inf` when called with finite
+            // values, it raises OverflowError instead.
+            Err($vm.new_overflow_error("math range error".to_owned()))
+        } else {
+            Ok($result)
+        }
+    }
+}
```

Comment:
Could this macro be replacable by an inline function? Unlike `make_math_func`, this macro looks like a funciton rather than a macro.

The code itself looks fine.

Reply:
Sounds good, does it look OK now?